### ISSUE TITLE
Updater prevents sleep mode

### ISSUE THREAD

**Issue by helptiger** [apparition47/MailTrackerBlocker#123]
Several times 11.4 doesn't go to sleep. It was blocked by a background task (updater) by your plugin.
Sorry, no further info's, I'm back on pre-updater version.

**Comment by apparition47** [comment]
Was able to reproduce this issue and spotted the hanging task:

```bash
$ pmset -g assertions | grep mailtrackerblocker
   pid 374(UserEventAgent): [0x00009d25000b89df] 00:00:04 BackgroundTask named: "com.oneefatgiraffe.mailtrackerblocker.updater"  
```

Will push out the fix for this next week. Appreciate your diagnosis once again 🙇‍♂️

**Comment by apparition47** [comment]
Fix released in 0.4.3, thanks @helptiger
